llvm/test/CodeGen/PowerPC/gcov_ctr_ref_init.ll

```diff
@@ -8,14 +8,19 @@ target triple = "powerpc-ibm-aix"
 
 ; CHECK-RW: .csect __llvm_covinit[RW],3
 ; CHECK-RO: .csect __llvm_covinit[RO],3
-; CHECK:         .vbyte  4, __llvm_gcov_writeout[DS]
-; CHECK-NEXT:    .vbyte  4, __llvm_gcov_reset[DS]
-; CHECK: __llvm_gcov_ctr.1:
+; CHECK-NEXT:    .align  3                               # @__llvm_covinit_functions
+; CHECK-NEXT: L..__llvm_covinit_functions:
+; CHECK-NEXT:     .vbyte  4, __llvm_gcov_writeout[DS]
+; CHECK-NEXT:     .vbyte  4, __llvm_gcov_reset[DS]
+; CHECK: L.._MergedGlobals:
```

Comment:
Can more context be added until we see the switch to the `__llvm_gcov_ctr_section`?

Reply:
Done! Now we are checking consecutive lines. Everything in the middle is `CHECK-NEXT` and we are not skipping any lines.